src/AspNet.Security.OAuth.Introspection/Events/AccessTokenReceivedContext.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.AspNetCore.Http;
+
+namespace AspNet.Security.OAuth.Introspection {
+    public class AccessTokenReceivedContext : BaseIntrospectionContext {
+        public AccessTokenReceivedContext(
+            HttpContext context, 
```

Comment:
Well, it has no runtime effect, but it's still a cool way to indicate which parameter is required which has basically no cost for us since we're referencing JetBrains.Annotations as a build-time dependency :smile: 

https://github.com/aspnet-contrib/AspNet.Security.OAuth.Extensions/blob/dev/src/AspNet.Security.OAuth.Introspection/project.json#L30